cloud/azure/bin/setup-keyvault

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env bash
+
+# TODO: figure out where in command flow to export PROJECT_BASE_DIR to
+# make it available in this file
+PROJECT_BASE_DIR="$(realpath "$(dirname "${BASH_SOURCE}")/../../../")"
+source "${PROJECT_BASE_DIR}/cloud/azure/bin/lib.sh"
+
+POSTGRES_PASSWORD_NAME="postgres-password"
+APP_SECRET_KEY_NAME="app-secret-key"
+
+# DOC: Create key vault instance, set permissions, and generate and set secrets
+
+while getopts g:v:l:u: flag
+do
+  case "${flag}" in
+    g) resource_group=${OPTARG};;
+    v) vault_name=${OPTARG};;
+    l) location=${OPTARG};;
+  esac
+done
+
+if [[ ! "${resource_group}" ]] \
```

Comment:
optional: you can mark these as readonly after the case.

https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names

Reply:
done